src/charm.py

```diff
@@ -571,6 +542,100 @@ def _gnb_subnet_config_is_valid(self) -> bool:
     def _get_gnb_subnet_config(self) -> Optional[str]:
         return self.model.config.get("gnb-subnet")
 
+    def _get_access_nad_config(self) -> Dict[Any, Any]:
+        return {
+            "cniVersion": "0.3.1",
+            "ipam": {
+                "type": "static",
+                "routes": [
+                    {
+                        "dst": self._get_gnb_subnet_config(),
+                        "gw": self._get_access_network_gateway_ip_config(),
+                    },
+                ],
+                "addresses": [
+                    {
+                        "address": self._get_access_network_ip_config(),
+                    }
+                ],
+            },
+            "capabilities": {"mac": True},
+        }
+
+    def _get_core_nad_config(self) -> Dict[Any, Any]:
+        return {
+            "cniVersion": "0.3.1",
+            "ipam": {
+                "type": "static",
+                "addresses": [
+                    {
+                        "address": self._get_core_network_ip_config(),
+                    }
+                ],
+            },
+            "capabilities": {"mac": True},
+        }
+
+    def _get_core_interface_mtu_config(self) -> Optional[str]:
```

Comment:
How about returning MTU as `int` right away? As `string` it's useless anyway.

Reply:
Done